amlip_cpp/include/amlip_cpp/types/status/NodeKind.hpp

```diff
@@ -42,18 +42,20 @@ enum class NodeKind
     main,
     computing,
     status,
-    meta
+    meta,
+    edge
 };
 
-const std::array<std::string, 7> NAMES_NodeKind =
+const std::array<std::string, 8> NAMES_NodeKind =
```

Comment:
This is an autogenerated file. Please, use the generator or modify this so it uses `ENUMARATION_BUILDER` (I think there was a limitation with SWIG, but it is worth the effort